CyberHealth/staticpages/templates/staticpages/accessibility-statement.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% block title %}Accessibility statement - Cyber health assessment{% endblock %}
+{% block content %}
+  <div class="govuk-width-container ">
+    <main class="govuk-main-wrapper govuk-body" id="main-content" role="main">
+      <h1 class="govuk-heading-xl">Accessibility statement</h1>
+
+      <p>
+        This statement applies to content published on the XXXXX domain. It does not apply to content on XXXXX subdomains.
+      </p>
+
+      <p>
+        This website is run by XXXXX. It is designed to be used by as many people as possible. The text should be clear and simple to understand. You should be able to:
```

Comment:
Could we change `website` to `web application` and could we say `useable` instead of `used`

Reply:
Could we also replace XXXXX with a link to the MHCLG?  and the text `MHCLG`?

Reply:
done in 0229fce / bfb0ec2